corehq/motech/dhis2/events_helpers.py

```diff
@@ -14,7 +14,19 @@ def send_dhis2_event(request, form_config, payload):
 
 
 def get_event(config, payload):
-    info = CaseTriggerInfo(None, None, None, None, None, form_question_values=get_form_question_values(payload))
+    info = CaseTriggerInfo(
+        domain=None,
+        case_id=None,
+        type=None,
+        name=None,
+        owner_id=None,
+        modified_by=None,
+        updates=None,
+        created=None,
+        closed=None,
+        extra_fields=None,
```

Comment:
Should `CaseTriggerInfo` provide defaults to make this a bitter shorter?

Reply:
Yes. I agree. I'm going to merge this to get the bug fix deployed. I have another branch that builds on this, and I'll rebase off master once this is in. I'll set defaults in that branch.